docs/graphql/manual/api-reference/schema-metadata-api/scheduled-triggers.rst

```diff
@@ -0,0 +1,309 @@
+   * - schedule
+     - true
+     - ScheduleConf_
+     - Schedule configuration
+   * - payload
+     - false
+     - JSON
+     - Any JSON payload which will be sent with the scheduled event
+   * - headers
+     - false
+     - [ HeaderFromValue_ | HeaderFromEnv_ ]
+     - List of headers to be sent with the webhook
+   * - retry_conf
+     - false
+     - RetryConfST_
+     - Retry configuration if scheduled event delivery fails
+
+
+.. _update_scheduled_trigger:
+
+update_scheduled_trigger
+------------------------
+
+``update_scheduled_trigger`` is used to update an existing scheduled trigger.
+
+.. _update_scheduled_trigger_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - name
+     - true
+     - TriggerName_
+     - Name of the scheduled trigger
+   * - webhook
+     - false
+     - String
+     - Full url of webhook
+   * - webhook_from_env
+     - false
+     - String
+     - Environment variable which has the full url of webhook
+   * - schedule
+     - true
+     - ScheduleConf_
+     - Schedule configuration
+   * - payload
+     - false
+     - JSON
+     - Any JSON payload which will be sent with the scheduled event
+   * - headers
+     - false
+     - [ HeaderFromValue_ | HeaderFromEnv_ ]
+     - List of headers to be sent with the webhook
+   * - retry_conf
+     - false
+     - RetryConfST_
+     - Retry configuration if scheduled event delivery fails
+
+.. _delete_scheduled_trigger:
+
+delete_scheduled_trigger
+------------------------
+
+``delete_scheduled_trigger`` is used to delete an existing scheduled trigger.
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+   {
+       "type" : "delete_scheduled_trigger",
+       "args" : {
+           "name": "sample_cron"
+       }
+   }
+
+.. _delete_scheduled_trigger_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - name
+     - true
+     - TriggerName_
+     - Name of the scheduled trigger
+
+.. _cancel_scheduled_event:
+
+cancel_scheduled_event
+----------------------
+
+``cancel_scheduled_event`` is used to cancel a particular run of a scheduled trigger.
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+   {
+       "type" : "cancel_scheduled_event",
+       "args" : {
+           "event_id": "237b604c-67f1-4aa8-8453-36855cfebfc4"
+       }
+   }
+
+.. _cancel_scheduled_event_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - event_id
+     - true
+     - UUID
+     - Id of the scheduled event
+ 
+.. _TriggerName:
+
+TriggerName
+&&&&&&&&&&&
+
+.. parsed-literal::
+
+  String
+
+.. _ScheduleConf:
+
+ScheduleConf
```

Comment:
@tirumaraiselvan I'm a little confused about the structure here. Does this still belong to the `cancel_scheduled_event` section? If not, shouldn't there be a new heading?

Reply:
@marionschleifer No, these are after `cancel_scheduled_event` section. This is beginning of syntax references that have been made above.